reliability/single-fit/rum/src/test/kotlin/com/datadog/android/rum/integration/RumConfigurationTest.kt

```diff
@@ -272,13 +279,143 @@ class RumConfigurationTest {
 
         // Then
         val eventsWritten = stubSdkCore.eventsWritten(Feature.RUM_FEATURE_NAME)
-        StubEventsAssert.assertThat(eventsWritten)
+        assertThat(eventsWritten)
             .hasRumEvent(index = 0) {
                 hasViewName(mappedViewName)
                 hasViewUrl(mappedViewUrl)
             }
     }
 
+    @RepeatedTest(16)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏ Do we really need repeated tests here and below ?

Reply:
Yes: because we don't have multiple use cases in the Integration tests, at least not enough to be exhaustive, we repeat the test to ensure there's enough variety in the randomised input to ensure a maximum coverage on the features we test